src/main/java/org/broadinstitute/hellbender/cmdline/CommandLineParser.java

```diff
@@ -71,6 +73,14 @@
     public static final String COMMENT = "#";
     public static final String POSITIONAL_ARGUMENTS_NAME = "Positional Argument";
 
+    // Map from (full class) name of each GATKCommandLinePluginDescriptor requested and
+    // found to the actual descriptor instance
+    private Map<String, GATKCommandLinePluginDescriptor<?>> pluginDescriptors = new HashMap<>();
+
+    // Return the plugin instance corresponding to the targetDescriptor class
+    public GATKCommandLinePluginDescriptor<?> getPluginDescriptor(Class<?> targetDescriptor) {
```

Comment:
I commented below, about this, but I think it would probably be better add a type parameter to this and have it cast the result internally to the target descriptor type.